hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterTrash.java

```diff
@@ -276,7 +276,7 @@ public void testMultipleMountPoint() throws IOException,
 
     // Client user see global trash view， wo should see all three mount point.
     FileStatus[] fileStatuses = fs.listStatus(new Path("/user/test-trash/.Trash/Current/"));
-    assertEquals(3, fileStatuses.length);
+    assertEquals(2, fileStatuses.length);
```

Comment:
Why does the size change?

Reply:
I add description in Description of PR. Thanks

Reply:
With https://github.com/apache/hadoop/pull/5039  , it will lead to user will not see any other nameservices trash path expect where the user home is mounted nameservices.    It will break https://issues.apache.org/jira/browse/HDFS-16024 purpose.